styles.css

```diff
@@ -1,3 +1,80 @@
-h1 {
-  color: #0ff;
+@media screen and (max-width: 768px) {
+  body {
+    background: #e5e5e5;
+    padding: 0;
+    margin: 0;
+  }
+
+  .headline {
+    background-image: url(icons/header-shapes\ mobile.png);
+    background-size: cover;
```

Comment:
- [ ] **[OPTIONAL]**  I think it would be better to make a small adjustment on your element spacing to replicate the Figma design and also the best user experience 😎😎

![image](https://user-images.githubusercontent.com/102757126/234974209-902ba97c-9bea-4747-a25b-a33653348012.png)

- [ ] **[OPTIONAL]** It would be better also to use the exact background image given in the Figma template design and kindly adjust it to fit the whole header section for the best user experience 🙏